staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -93,6 +93,13 @@ type KubeSchedulerConfiguration struct {
 	// with the extender. These extenders are shared by all scheduler profiles.
 	// +listType=set
 	Extenders []Extender `json:"extenders,omitempty"`
+
+	// TryNominatedNodeFirst tells scheduler whether it is allowed to short-circuit to check the
+	// nominated node directly instead of loop over all the nodes in the cluster.
+	// Set this to true implies the preemptor pod might not be dispatched to the best candidate in
+	// some corner case, e.g. another node releases the resources and hence is the best candidate
+	// while the victims got removed from the nominated node.
+	TryNominatedNodeFirst bool `json:"tryNominatedNodeFirst,omitempty"`
```

Comment:
Let's discuss the need of this option. Could you add an entry for the next SIG meeting?

Reply:
I will check the schedule and see whether I can attend in person.

Reply:
Added a topic in a agenda, it's a little late for me but I will try my best to attend.

Reply:
I haven't looked at the details of the PR yet, but +1 to not having this as a config flag. We could use a feature flag to roll this out, but a config flag isn't useful.

Reply:
Done